File: src/main/java/MachTeacher/MachTeacher/dto/AuthResponse.java
package MachTeacher.MachTeacher.dto;

import lombok.Getter;
import lombok.Setter;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class AuthResponse {
    private String token;
    private Long id;
    private String fullName;
    private String email;
    private String role;

    private String phone;
    private String city;
    private String country;

    private String university;
    private String career;
    private String semester;
}

File: MachTeacher/app/src/main/java/com/example/machteacher/dto/AuthResponse.kt
package com.example.machteacher.dto


data class AuthResponse(
    val token: String,
    val id: Long,
    val fullName: String,
    val email: String,
    val role: String,
    val phone: String?,
    val city: String?,
    val country: String?,
    val university: String?,
    val career: String?,
    val semester: String?
)
